.circleci/config.yml

```diff
@@ -160,6 +189,13 @@ workflows:
               only:
                 - main
       - firebase-robo:
```

Comment:
I noticed that nothing is dependent on this. We should make publishing the demo app and the SDK dependent on this passing.

Reply:
Ok!